modules/2d_inference_3d_volume.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<p>Please the empty cell.</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>